.eslintrc.js

```diff
@@ -22,7 +23,6 @@ module.exports = {
     },
     sourceType: 'module',
   },
-  plugins: ['babel', 'react', 'flowtype', 'import', 'prettier'],
```

Comment:
Question: Why did this move locations?

Reply:
Ah I changed more things in this file before, and I thought it was a good idea to have `plugins` above the `extends` that reference them. This should make no difference at all.